packages/design-system/src/atoms/buttonV2/index.tsx

```diff
@@ -39,6 +39,8 @@ const BaseButton = ({
   namedIcon,
   isDisabled = false,
   isLoading = false,
+  colorIcon,
+  backgroundColorIcon,
```

Comment:
Let's make the naming English-friendly :D

```suggestion
  iconColor,
  iconBackgroundColor,
```